common/djangoapps/entitlements/helpers.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, timedelta
+
+import pytz
+import sys
+from django.conf import settings
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+def is_entitlement_expired(entitlement):
+    """
+    Determines from the policy if an entitlement can be redeemed, if it has not passed the
+    expiration period of policy.expiration_period_days, and has not already been redeemed
```

Comment:
It might be clearer to name this `can_be_redeemed` or something like that, as this method may return `False` even though the entitlement should be expired (if, for instance, it's already been redeemed).

Reply:
:+1: